cmd/mattermost/commands/message_export.go

```diff
@@ -79,3 +106,57 @@ func messageExportCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func csvExportCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	startTime, err := command.Flags().GetInt64("exportFrom")
+	if err != nil {
+		return errors.New("exportFrom flag error")
+	} else if startTime < 0 {
+		return errors.New("exportFrom must be a positive integer")
+	}
+
+	if a.MessageExport == nil {
+		CommandPrettyPrintln("MessageExport feature not available")
+	}
+
+	err2 := a.MessageExport.RunExport("csv", startTime)
+	if err2 != nil {
+		return err2
+	}
+	CommandPrettyPrintln("SUCCESS: Your data was exported.")
+
+	return nil
+}
+
+func actianceExportCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	startTime, err := command.Flags().GetInt64("exportFrom")
+	if err != nil {
+		return errors.New("exportFrom flag error")
+	} else if startTime < 0 {
+		return errors.New("exportFrom must be a positive integer")
+	}
+
+	if a.MessageExport == nil {
+		CommandPrettyPrintln("MessageExport feature not available")
+	}
+
+	err2 := a.MessageExport.RunExport("actiance", startTime)
+	if err2 != nil {
+		return err2
+	}
+	CommandPrettyPrintln("SUCCESS: Your data was exported.")
+
+	return nil
```

Comment:
Only 1/20 of the LOC of these two functions are different. Do you think it merits some de-duplication?

Reply:
Fair point, I've changed it. Confirm me that you like it.